rpc-jobs/RPC-AIO.yml

```diff
@@ -74,13 +74,14 @@
       - tempest_params
       - string:
           name: STAGES
-          default: "Allocate Resources, Connect Slave, Prepare Deployment, Deploy RPC w/ Script, Install Tempest, Tempest Tests, Cleanup"
+          default: "Allocate Resources, Connect Slave, Prepare Deployment, Prepare Configs, Deploy RPC w/ Script, Install Tempest, Tempest Tests, Cleanup"
```

Comment:
There isn't a Prepare Configs stage anymore since it got moved back into Prepare Deployment.